flags.md

```diff
@@ -0,0 +1,141 @@
+```nim
+
+## From build.nims
+
+# Arch:
+
+--cpu:arm                             # tags: arm32
+--cpu:arm64                           # tags: arm64
+--cpu:i386                            # tags: x86
+
+--define:bit32                        # tags: arm32
+--define:bit32                        # tags: x86
+
+--gcc.path:/usr/bin                   # tags: arm64
+--gcc.exe:aarch64-linux-gnu-gcc       # tags: arm64
+--gcc.linkerexe:aarch64-linux-gnu-gcc # tags: arm64
+
+--passC:'-m32'                        # tags: x86, gcc
+--passL:'-m32'                        # tags: x86, gcc
+
+# Developer
+
+--embedsrc:on                         # tags: dev
+--define:DEV                          # tags: dev
+--listCmd                             # tags: dev
+
+--define:DEBUG                        # tags: debug
+--debugger:on                         # tags: debug
+--debuginfo                           # tags: debug
+--linedir:on                          # tags: debug
+
+--define:PROFILE                      # tags: memprofile
+--profiler:off                        # tags: memprofile
+--stackTrace:on                       # tags: memprofile
+--define:memProfiler                  # tags: memprofile
+
+--define:PROFILE                      # tags: profiler
+--profiler:on                         # tags: profiler
+--stackTrace:on                       # tags: profiler
+
+--debugger:native                     # tags: profilenative
+
+# Compilation
+
+--define:OPTIMIZED                    # tags: optimized
+
+--define:strip                        # tags: release
+--passC:-flto                         # tags: release, unix
+--passL:-flto                         # tags: release, unix
+
+# Compilation: full
+
+--define:ssl                          # tags: full
+
+# Compilation: mini/web
+
+--define:SAFE                         # tags: safe
+--define:MINI                         # tags: mini
+--define:WEB                          # tags: web
+
+--verbosity: 3                        # tags: web
```

Comment:
The idea of the new implementation is to `--verbosity` depend on the `--who` parameter.
By default, user should have `--verbosity:0`, while developers `--verbosity:3`.

* Should this be always 3 for web-builds?

Reply:
As a side-comment: web builds are generally *very* problematic. That's why it was set to `3`...

Reply:
Alright!